lib/shippo/api.rb

```diff
@@ -44,9 +44,7 @@ def debug?
   end
 end
 
-Shippo.dir('shippo/api')
-Shippo.dir('shippo/api/transformers')
-Shippo.dir('shippo/api/extend')
-Shippo.dir('shippo/model')
-
-
+Dir[File.dirname(__FILE__) + '/api/*.rb'].each {|file| require file }
```

Comment:
@kigster the issue was 
```
cannot load such file -- require_dir
```
on lib/shippo.rb line 11. I tried including require_dir gem in my own project, trying requiring it before require shippo, and tried a bunch of things. It worked perfectly in development but not on staging/production like environment. I didn't had too much time to dig deeper unfortunately, but removing it completely solved the issue and the gem is working properly.

I don't think requiring the way I did is a lot of repetitive code, in fact I reduced the number of lines, and it's a nice trade off not to add another dependency. Other ruby projects usually prefer not requiring all files in the dir to keep it consistent and easy to debug. We can also consider that, but it's more lines of code.